jax/lax/lax.py

```diff
@@ -1731,6 +1739,23 @@ def betainc_grad_not_implemented(g, a, b, x):
 
 digamma_p = standard_unop(_float, 'digamma')
 
+igamma_p = standard_naryop([_float, _float], 'igamma')
+
+def igamma_gradx(g, a, x):
+  return g * exp(-x + (a - 1.) * log(x) - lgamma(a))
+
+def gamma_grad_not_implemented(a, b, x):
+  raise ValueError("Igamma(c) gradient with respect to `a` not supported.")
```

Comment:
Out of curiosity, what would we need to implement this? Leave a TODO?

Reply:
Added a TODO. We could reuse some of the gamma reparameterization code in the gamma sampler, but I'm also planning on exposing an Igamma_Grad_A op in XLA (which will also allow the corresponding TF code to be XLA compiled).